recipes/snippy/meta.yaml

```diff
@@ -25,8 +25,8 @@ requirements:
     - bedtools >=2.28.0
     - parallel >=20170422
     - freebayes >=1.3.1
-    - vcflib >=1.0.0_rc3
-    - snpeff >=4.3
+    - vcflib >=1.0.0_rc3,<=1.0.2
```

Comment:
Can you please add a comment why the lower pins are needed?

Reply:
Running snippy with vcflib version 1.0.3, starting from build `h6b7c446_3` returns an error: [error_log.txt](https://github.com/bioconda/bioconda-recipes/files/12144761/snippy_error_vcflib_h6b7c446_3.txt). Version 1.0.3, build `ha04fe3b_2` and lower do not return this error. AFAIK it's not possible to specify "lower than" operators for builds, so I've constained the vcflib version here to 1.0.2 and lower to avoid build `h6b7c446_3` and later for version 1.0.3.

The SnpEff version is constrained due to this error which occurs starting with SnpEff version 5.1, when providing a reference genome annotated in Genbank format (line 436 has the error): [error_log.txt](https://github.com/bioconda/bioconda-recipes/files/12144795/snippy_error_snpeff5.1.txt)

Reply:
Thanks for the explanation. Can you please add a short version of this into the meta.yaml file? Thanks

Reply:
Ah sorry, I've added it in the meta.yaml now

Reply:
Cool!